Source/WebCore/platform/graphics/ImageBuffer.h

```diff
@@ -163,6 +163,8 @@ class ImageBuffer : public ThreadSafeRefCounted<ImageBuffer, WTF::DestructionThr
     void setBackendInfo(ImageBufferBackend::Info&& parameters) { m_backendInfo = WTFMove(parameters); }
 
     WEBCORE_EXPORT virtual RefPtr<NativeImage> copyNativeImage(BackingStoreCopy = CopyBackingStore) const;
+    WEBCORE_EXPORT virtual RefPtr<NativeImage> copyVolatileNativeImage(BackingStoreCopy = CopyBackingStore) const;
```

Comment:
It's odd for these to be here on ImageBuffer. As a caller, I would not know how to pick which function to call. What's "volatile" from the perspective of the caller?

If we need to expose something here, I'd suggest an OptionSet<> or enum that communicates what we need to know (like, "this is a short-lived image" or whatever). Or we change the API like we did with `sinkIntoImage`.